jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile

```diff
@@ -58,6 +83,24 @@ pipeline {
                 }
             }
         }
+        stage('Upload Artifacts to Sonatype') {
+            tools {
+                maven "maven-3.8.2"
+            }
+            environment {
+                REPO_URL = "https://aws.oss.sonatype.org/"
+                STAGING_PROFILE_ID = "${SONATYPE_STAGING_PROFILE_ID}"
+                BUILD_ID = "${BUILD_ID}"
+            }
+            steps {
+                script {
+                    // stage artifacts for release with Sonatype
+                    withCredentials([usernamePassword(credentialsId: 'Sonatype', usernameVariable: 'SONATYPE_USERNAME', passwordVariable: 'SONATYPE_PASSWORD')]) {
+                        sh('$WORKSPACE/publish/stage-maven-release.sh maven')
```

Comment:
Does the first parameter need to be the absolute path like provided in the example?

https://github.com/opensearch-project/opensearch-build/blob/38289d8c8fcb4cdc1118a3c730cf489dd469ecc1/publish/stage-maven-release.sh#L17